models/article.model.js

```diff
@@ -0,0 +1,49 @@
+const { query } = require('../db/connection');
+const db = require('../db/connection');
+const format = require('pg-format');
+
+exports.selectTopics = async () => {
+	let queryStr = `SELECT * FROM topics`;
+
+	const result = await db.query(queryStr);
+	return result.rows;
+};
+
+exports.selectArticleID = (article_id) => {
+	return db
+		.query('SELECT * FROM articles WHERE article_id = $1;', [article_id])
+		.then(({ rows }) => {
+			if (rows.length === 0) {
+				return Promise.reject({ msg: 'Article not found', status: 404 });
+			}
+			return rows[0];
+		});
+};
+
+exports.updateArticleID = (voteUpdate, id) => {
+	const { inc_votes } = voteUpdate;
+	const voteUpdateAmmount = voteUpdate.inc_votes;
+	if (typeof voteUpdateAmmount !== 'number') {
+		return Promise.reject({
+			status: 422,
+			msg: 'Unprocessable Entity- Please provide in format inc_votes: vote_number',
+		});
+	}
+	return db
+		.query('SELECT votes FROM articles WHERE article_id = $1 ;', [id])
+		.then(({ rows }) => {
+			let currentVote = rows[0].votes;
+			let newVote = (currentVote += voteUpdateAmmount);
+			return newVote;
+		})
+		.then((newVote) => {
+			return db
+				.query(
+					'UPDATE articles SET votes =$1 WHERE article_id =$2 RETURNING *;',
```

Comment:
All good, just watch the spaces around your = here

Reply:
Thank you so much will update accordingly :)